opentelemetry_collector/run.sh

```diff
@@ -1,21 +1,36 @@
 #!/bin/bash
 
 BUILD_DATE_FILE=/config/build_date.txt
```

Comment:
What format is the build date in this file? I wonder whether we can use an env var as well.